### ISSUE TITLE
[Bug] import statement in method: 'error parsing code, unable to find "debug" function statement'

### ISSUE THREAD

**Issue by Jasha10** [samuelcolvin/python-devtools#84]
# Bug

When the `from devtools import debug` statement occurs within a method (as opposed to at toplevel of the module), an "error parsing code" message is printed.

# Min repro:

The following works as expected:
```python
# tmp.py
from devtools import debug
def foo():
    debug(123)
foo()
```
```text
$ python tmp.py
tmp.py:4 foo
    123 (int)
```
But if you put the import statement inside the method, there is an error:

```python
# tmp2.py
def foo():
    from devtools import debug
    debug(123)
foo()
```
```text
$ python tmp2.py
tmp2.py:4 foo (error parsing code, unable to find "debug" function statement)
    123 (int)
```

**Comment by Jasha10** [comment]
Version info:
```text
$ pip list | grep devt
devtools                      0.6.1
$ python --version
Python 3.9.5
```

**Comment by samuelcolvin** [comment]
ye, to find the text inside the parentheses `debug('this bit')`, devtools has to search the string from the line where `debug()` is called.

That means sometimes it finds the wrong reference to `debug()`. If you can find a fix, happy to accept, but I don't have time to work on this right now.

**Comment by alexmojaki** [comment]
This is fixed in #82

**Comment by samuelcolvin** [comment]
Good point, I had a vague memory that this should be fixed. I'll look at that soon.

**Comment by samuelcolvin** [comment]
fixed by #82
